C-Plus-Plus/ds/AppendLastNToFirst - Linked List.cpp

```diff
@@ -0,0 +1,98 @@
+/* 
+"Copyright [2021] <Anshika Dubey>"
+You have been given a singly linked list of integers along with an integer 'N'. Write a function to append the last 'N' nodes towards the front of the singly linked list and returns the new head to the list.
```

Comment:
The description can be cut down to multiple lines. 
After every full stop, bring the sentence to the next line. Improves readablity.